packages/components/src/dropdown-menu/index.js

```diff
@@ -71,6 +71,7 @@ function DropdownMenu( {
 						labelPosition={ __unstableLabelPosition }
 						tooltip={ label }
 					>
+						{ ! icon && <span className="components-dropdown-menu__label" > { label } </span> }
```

Comment:
This change has an impact on a lot of block toolbars (paragraph for instance) We probably shouldn't use the label in a generic way like that as it looks like a breaking change compared to the previous behavior.

Reply:
yes you are right of course :)